src/test/java/de/retest/web/it/ShowcaseIT.java

```diff
@@ -1,16 +1,20 @@
 package de.retest.web.it;
 
+import static de.retest.web.testutils.PageFactory.page;
+import static de.retest.web.testutils.PageFactory.Page.SHOWCASE;
+
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 import org.openqa.selenium.WebDriver;
 
 import de.retest.recheck.Recheck;
 import de.retest.recheck.RecheckImpl;
-import de.retest.web.testutils.PageFactory;
-import de.retest.web.testutils.PageFactory.Page;
+import de.retest.recheck.junit.jupiter.RecheckExtension;
 
+@ExtendWith( RecheckExtension.class )
```

Comment:
Since this is a showcase, should we add something like:
`// Use extension to administer lifecycle of test`